test/json_post_logger.ts

```diff
@@ -36,7 +30,7 @@ test('log via post payload request to example.com', async (done) => {
     { type: 'json_post', host: 'logging.example.com' },
   ]);
 
-  const loggingEndpoint = nock('http://logging.example.com')
+  const loggingEndpoint = nock(/example.com/)
```

Comment:
instead of using /example.com/ can you use a bombbomb url for safety

Reply:
Example is RFC'd for this purpose plus this library is open source so it's nice not to tie it so directly to our properties.